quiz/urls.py

```diff
@@ -0,0 +1,9 @@
+from django.urls import path
+from .views import *
+
+
+urlpatterns = [
+    path('', index),
+    path('quiz/<int:pk>/', get_quiz_by_pk, name='get_quiz'),
+    path('submit-quiz/<int:pk>/', submit_quiz, name='submit_quiz')
```

Comment:
As you can see, the quiz GET url and the quiz POST url are very similar. You can use the same url for both and check in the view the request.method to differentiate between GET and POST code.

In many cases a part of the code is the same for both (at least the check for 404) so this is one of the reasons why usually the same url is used for both.